cmd/help.go

```diff
@@ -13,7 +13,7 @@ import (
 func HelpTemplate() string {
 	cobra.AddTemplateFunc("now", time.Now)
 	cobra.AddTemplateFunc("caller", getCaller)
-	return `© {{ now.Year }} Appgate Cybersecurity, Inc.
+	return `© Appgate Cybersecurity, Inc.
```

Comment:
```suggestion
	return `© 2022-{{ now.Year }} Appgate Cybersecurity, Inc.
```
This is better.

Reply:
it's incorrect. Copyright is about when it was written, not when it's being executed.

Reply:
It's also supposed to guide when the copyright may expire, which it doesn't do if you remove the year entirely. I don't think I've seen a copyright like this without a year provided.

Reply:
How can it guide on when the copyright expires if it always shows today's date?

Reply:
Just leave `2022` there, and an indicator that is wrong for a short time every year is absolutely better than nothing at all

Reply:
actually, just print 2022-2023. By the time we release the next version, it's going to be correct